package.json

```diff
@@ -36,17 +36,17 @@
     "lint": "tslint src/**/*.ts",
     "install": "node scripts/install.js",
     "postinstall": "node scripts/post-install.js",
-    "test": "cross-env NODE_ENV=test mocha -R spec lib/*.test.js",
+    "test": "cross-env NODE_ENV=test mocha -R spec --exit lib/*.test.js",
     "prepublish": "npm run tsc"
   },
   "dependencies": {
     "nan": "^2.6.2"
   },
   "devDependencies": {
-    "@types/mocha": "^2.2.41",
-    "@types/node": "^6.0.58",
-    "cross-env": "^3.2.4",
-    "mocha": "^3.1.2",
+    "@types/mocha": "^5.0.0",
+    "@types/node": "^9.6.2",
```

Comment:
Updating node types will make it easy to accidentally break old node support without knowing.

Reply:
should I leave node LTS instead of latest ? Is just @types/node right ? IMO, node 6 is too outdated but perhaps you want me to leave it as it was ? please advice and I will me the chage, thanks! :)

Reply:
there you have it :  "@types/node": "^6.0.104" hope it's fine, good job with this library!!! :)